server/tests/io/feed_parsers/belga_newsml_1_2_test.py

```diff
@@ -212,18 +222,6 @@ def test_content(self):
         self.assertEqual(item["source"], "ANP/BELGA")
         self.assertEqual(item["extra"], {'city': 'ANTWERPEN'})
         self.assertEqual(item["type"], "text")
-        self.assertEqual(
```

Comment:
imo it would be good to check it's actually not there, eg. `self.assertNotIn('abstract', item)`

Reply:
can we do this `self.assertEqual(item['abstract'], '')` as we will have a key in the item but it will be always empty?

Reply:
yep that's ok